tools/gromacs/setup.xml

```diff
@@ -9,13 +9,11 @@
     <command detect_errors="exit_code"><![CDATA[
 
         ln -s '$pdb_input' ./pdb_input.pdb &&
```

Comment:
to we need to bump the tool version here?

Reply:
done